### PR TITLE
Update global styles and add styling for new pages

### PR DESCRIPTION
**Pull request by mattlangan**
This commit introduces a number of global changes in both DOM and CSS.

Global changes:

- Updated global color variables and USWDS H* font-size declarations to reflect new cohesive style guide
- Updated all button styles and made the PDF button more congruent with the Info button
- Removed custom nav section styles and added hover-state with underline and color change
- Added hover transitions to all buttons, including accordions and navigation
- Removed "Home" link and icon from breadcrumbs (This will probalby need to be updated again in the future, as I left the `<a>` tag in place, just without any href value, and put a non-breaking space in its content)
- Replaced Poppins with Open Sans as primary typeface and removed secondary typeface
- Moved font import from `<head>` href to an `@import` in `main.scss` to keep CSS-related stuff in 1 place

Footer:

- Introduced 4-column layout, with a "Quick Links" list on the left for dispatch to key pages
- Added "PrimaryFooter" Menu in Contentful to power Quick Links list
- Removed tiered footer links in favor of a single footer link style
- Moved Promo/Social Proof section to beneath footer links

Updated page UI:

- Updated Calendar UI
- Updated Press Release UI
- Updated Contacts UI
- Updated Room UI
- Sub-page/Sub-section now appear beneath content blocks

### CODE REVIEW

File: _assets/stylesheets/components/breadcrumbs.scss [hunk 1]
```diff
@@ -36,13 +36,12 @@
...
 
 .acc-breadcrumb-home {
-  background-image: image_url('icons/globe.svg');
   background-position: center left;
   background-repeat: no-repeat;
   background-size: 1.6em;
   padding: 0;
   text-indent: -9999px;
-  width: 2.4em;
+  cursor: default;
```
Comment (jcbpl) [inline _assets/stylesheets/components/breadcrumbs.scss]: Doesn't this just make the globe disappear? Why is cursor: default necessary?

File: _assets/stylesheets/components/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.acc-event-title {
+  margin-bottom: .5rem;
+  margin-top: 0;
+}
+
+.acc-calendar-event {
+  border-bottom: 1px solid $acc-color-gray-light;
+  margin-bottom: 3rem;
+  padding-bottom: 1em;
+}
```
Comment (jcbpl) [inline _assets/stylesheets/components/calendar.scss]: What's the difference between .acc-event and .acc-calendar-event?

File: _assets/stylesheets/elements/reset.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+* {
+	-webkit-margin-start: 0;
+	-webkit-margin-end: 0;
+	-webkit-margin-before: 0;
+	-webkit-margin-after: 0;
+}
```
Comment (jcbpl) [inline _assets/stylesheets/elements/reset.scss]: Why is this necessary?

File: _includes/analytics/google_analytics.html [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+  87703455-1 Profile: "ACC Alpha"
+-->
 <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-  ga('create', 'UA-83539778-3', 'auto');
+  ga('create', 'UA-87703455-1', 'auto');
```
Comment (jcbpl) [inline _includes/analytics/google_analytics.html]: I think we should exclude analytics snippets in the development env.

File: _includes/components/content_blocks/text.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {% assign block = include.block %}
 
-<h3>{{ block.title }}</h3>
+<!-- <h3>{{ block.title }}</h3> -->
```
Comment (jcbpl) [inline _includes/components/content_blocks/text.html]: What's up with this? We should definitely use/render the title.

File: _assets/stylesheets/components/breadcrumbs.scss (hunk 1 above)
Comment (mattlangan) [inline _assets/stylesheets/components/breadcrumbs.scss]: @jcbpl In the long-run the <a> empty tag should probably be replaced by a <span> or other element, but for now the cursor change is to prevent the hover state from making it appear as though it's a clickable element to the left of the first breadcrumb.

File: _assets/stylesheets/elements/reset.scss (hunk 3 above)
Comment (mattlangan) [inline _assets/stylesheets/elements/reset.scss]: @jcbpl I forget which element it was applying to, but webkit was indenting some block-level element(s) through its default styling. It was just the browser that was applying the property, nothing from USWDS or ACC styling.

File: _includes/analytics/google_analytics.html (hunk 4 above)
Comment (mattlangan) [inline _includes/analytics/google_analytics.html]: @jcbpl Localhost shouldn't be sending GA data unless it's specified as a site URL in GA. The goal is to gather analytics data during alpha.

File: _assets/stylesheets/components/breadcrumbs.scss (hunk 1 above)
Comment (jcbpl) [inline _assets/stylesheets/components/breadcrumbs.scss]: I don't understand; do we not want the globe there at all? We should just remove it if that's the case.

File: _includes/analytics/google_analytics.html (hunk 4 above)
Comment (jcbpl) [inline _includes/analytics/google_analytics.html]: Ok. It still fetches ga.js on hard refresh, which is a bit annoying, but we can change it in a separate commit.

File: _includes/components/content_blocks/text.html (hunk 5 above)
Comment (jcbpl) [inline _includes/components/content_blocks/text.html]: We need the title as a separate attribute for accordion blocks, so duplicating it in the body just makes it appear twice in accordions. I'm almost certain we could find a good way to specifically target the elements in the room views to do whatever we need.
